notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -532,11 +532,11 @@ export default {
     },
     addNote() {
       if (!this.hasDraft) {
-        window.open(`${eXo.env.portal.context}/${eXo.env.portal.portalName}/notes-editor?spaceId=${eXo.env.portal.spaceId}&parentNoteId=${this.note.id}&appName=${this.appName}`, '_blank');
+        window.open(`${eXo.env.portal.context}/${eXo.env.portal.portalName}/notes-editor?spaceId=${eXo.env.portal.spaceId}&parentNoteId=${this.note.id}&spaceGroupId=${eXo.env.portal.spaceGroup}&appName=${this.appName}`, '_blank');
```

Comment:
Could you add a nullity check for `eXo.env.portal.spaceGroup` to avoid having a null value ?